kolibri/plugins/coach/assets/src/views/quizzes/QuizSummaryPage/QuizOptionsDropdownMenu.vue

```diff
@@ -41,7 +41,7 @@
           },
           { label: this.coreString('deleteAction'), value: 'DELETE' },
         ];
-        if (!this.exam?.archive) {
+        if (this.exam?.data_model_version > 2) {
```

Comment:
Would `3` represent our most current exam?

Reply:
Yes and it would means it comes after DraftExam existed.

Reply:
For purposes of clarity, I think let's create a computed prop with a descriptive name as this may not be clear to everyone, particularly those those that are less familiar with how exams operate. I would also make it more explicit and use `this.exam?.data_model_version >= 3` instead as the current version is known, and its also just easier to understand at a glance (optional). 

Question, is there a time when `data_model_version` is empty? Looks to me that the model storing this value has a default? If not, then we don't need the chaining.